content/3_Silver/Binary_Search.mdx

```diff
@@ -0,0 +1,1047 @@
+---
+id: binary-search
+title: 'Binary Search'
+author: Darren Yao, Abutalib Namazov, Andrew Wang
+prerequisites:
+  - intro-greedy
+description: 'Binary searching on arbitrary monotonic functions and built-in functions for binary search.'
+frequency: 3
+---
+
+export const problems = {
+	ex: [
+		new Problem(
+			'CF',
+			'Div 2 C - Maximum Median',
+			'contest/1201/problem/C',
+			'Easy',
+			false,
+			['Binary Search'],
+			''
+		),
+	],
+	usaco: [
+		new Problem(
+			'Silver',
+			'Cow Dance Show',
+			'690',
+			'Easy',
+			false,
+			['Binary Search', 'Ordered Set'],
+			'usaco-690'
+		),
+		new Problem(
+			'Silver',
+			'Convention',
+			'858',
+			'Easy',
+			false,
+			['Binary Search'],
+			'usaco-858'
+		),
+		new Problem(
+			'Silver',
+			'Angry Cows',
+			'594',
+			'Easy',
+			false,
+			['Binary Search'],
+			'usaco-594'
+		),
+		new Problem(
+			'Silver',
+			'Social Distancing',
+			'1038',
+			'Normal',
+			false,
+			['Binary Search'],
+			'usaco-1038'
+		),
+		new Problem(
+			'Gold',
+			'Angry Cows',
+			'597',
+			'Hard',
+			true,
+			['Binary Search'],
+			''
+		),
+		new Problem(
+			'Silver',
+			'Loan Repayment',
+			'991',
+			'Very Hard',
+			false,
+			['Binary Search', 'Sqrt'],
+			'requires some rather tricky analysis to speed up naive $mathcal{O}(Nlog N)$ solution'
```

Comment:
\log

Reply:
this doesn't actually appear anywhere so it doesn't matter :P